pkg/services/datasources/models.go

```diff
@@ -92,6 +93,20 @@ func GetTeamHTTPHeaders(jsonData *simplejson.Json) (TeamHTTPHeaders, error) {
 		if err != nil {
 			return nil, err
 		}
+		for teamID, headers := range teamHTTPHeadersJSON {
+			if teamID == "" {
+				return nil, errors.New("teamID is missing or empty in teamHttpHeaders")
+			}
+
+			for _, header := range headers {
+				if header.Header == "" {
```

Comment:
I think we can simplify this a bit and just say that header is empty.
```suggestion
				if header.Header == "" || header.Value == "" {
```

Reply:
i think we disagree here, i like the explicit nature of saying the value or the header. then we know which was was missing